### PR TITLE
Required updates for optional Web Services

### PR DESCRIPTION
**Pull request by thecodefactory**
Depends on https://github.com/libbitcoin/libbitcoin-server/pull/449

### CODE REVIEW

File: generate.xml [hunk 1]
```diff
@@ -1875,6 +1883,9 @@
         <library name="bitcoin-protocol" />
         <library name="bitcoin-node" />
         <headers path="include" />
+        <!-- for mbedtls and mongoose C project support -->
+        <headers path="src/external/mbedtls/include" />
+        <option value="CFLAGS='-std=c99 -Wno-pedantic -Wno-format -Wno-comment -Wno-long-long -Wno-overlength-strings'" />
```
Comment (evoskuil) [inline generate.xml]: This seems pretty onerous to apply to the full server build. Is it the case that these *only* affect the build of .c files?

Comment (thecodefactory) [reply generate.xml]: Yes, I tested this many times in order to get right.  It only affects CFLAGS and not the CPPFLAGS.

Comment (evoskuil) [reply generate.xml]: Cool

**Comment by evoskuil** [comment]
Mongoose dependency rejected due to licensing.

**Pull request closed without merging**